test/e2e/e2e_test.go

```diff
@@ -42,7 +42,7 @@ var (
 
 func TestEndToEnd(t *testing.T) {
 	RegisterFailHandler(Fail)
-	SetDefaultEventuallyTimeout(1 * time.Minute)
+	SetDefaultEventuallyTimeout(5 * time.Minute)
```

Comment:
My preference would be to keep the default timeout at 1 minute, and extend it if necessary for exceptional cases. It should be pretty rare that we really need 5 minutes to satisfy a test expectation.